docs/source/basic_usage.rst

```diff
@@ -254,3 +254,20 @@ interact with it. Even better, you can **go back in time** because you know what
     vdb <your_json_report>
 
 For detailed commands, please refer to :doc:`virtual_debug`
+
+Compress Your Report
+--------------------
+
+VizTracer supports compressing your json report. The general compress ratio is about 1% - 3% for a large report.
+
+You can compress your report with ``--compress``.
+
+.. code-block:: 
+
+    viztracer -o result.cvf --compress result.json
```

Comment:
I think maybe ``viztracer --compress result.json -o result.cvf`` makes more sense to the users. Compress then set the output file, compared to set the output file then compress.